### ISSUE TITLE
Stronger Paperspace support

### ISSUE THREAD

**Issue by AntonioCiolino** [TheLastBen/fast-stable-diffusion#101]
I see that there are a few posts here about how to make paperspace operate with the speedups, however, I don't see a way to make it operate without doing some deeply technical scripting, etc. I also did try to get the compiled files into paperspace, but the CPUs + the version of cuda didn't really support the A4000, and jamming the assembly files plus whatever override you have in your colab book to remove the safety checker didn't seem to work smoothly with their systems.

With all that said, it seems that paperspace does have a good methodology and great support for GPU runs, and it's fast. It doesn't do as well with filesystem support, but it does churn out images. 

If you ahve time, could you figure out a way to enable Paperspace support with automatic1111 better than what exists now?

**Comment by TheLastBen** [comment]
I'll give it a try

**Comment by AntonioCiolino** [comment]
Looks like RTX4000 and P5000 are generally freely available; lmk if I can be of help here by running anything or testing.

**Comment by dhsusf** [comment]
This would be huge for me personally if I could run on paperspace. I literally used my Google Colab credits on the Pro plan in less than 2 days (for other GPU intensive tasks).

**Comment by AntonioCiolino** [comment]
I did too. I’m using voldemort right now, modified to three cells.-ACOn Oct 22, 2022, at 4:48 PM, dhsusf ***@***.***> wrote:﻿
This would be huge for me personally if I could run on paperspace. I literally used my Google Colab credits on the Pro plan in less than 2 days (for other GPU intensive tasks).

**Comment by timothepearce** [comment]
@dhsusf @AntonioCiolino did you find any notebook I can easily run on Paperspace?

**Comment by AntonioCiolino** [comment]
Yes. There’s one I built but there’s also a much better one that’s been good at engineer-of-stuff GitHub space.-ACOn Oct 28, 2022, at 6:50 AM, Timothé Pearce ***@***.***> wrote:﻿
@dhsusf @AntonioCiolino did you find any notebook I can easily run on Paperspace?

**Comment by dhsusf** [comment]
@AntonioCiolino I'm also using the engineer-of-stuff notebook. I subscribed to the Growth plan on paperspace so now I'm using Joe Penna's library for training and then engineer-of-stuff for automatic1111. It's pretty smooth right now.

**Comment by AntonioCiolino** [comment]
Joe penna…haven’t seen that yet. How is training going? Any tips?-ACOn Oct 28, 2022, at 12:55 PM, dhsusf ***@***.***> wrote:﻿
@AntonioCiolino I'm also using the engineer-of-stuff notebook. I subscribed to the Growth plan on paperspace so now I'm using Joe Penna's library for training and then engineer-of-stuff for automatic1111. It's pretty smooth right now.

**Comment by dhsusf** [comment]
@AntonioCiolino Honestly, I just know it's one of the libraries that requires a lot of GPU Ram versus the Google Colab versions, but I don't really know how the end result compares. But the workflow is the same, so the experience isn't any difference.

In terms of tips, this if very use case specific, but it seems like Asian faces require a lot more training images and steps than others for the Dreambooth application. Well, at least that's my impression because on the stablediffusion reddit you read about people training on 15-20 images for 1500-2000 steps and showing impressive results. I tried that for a week and kept getting poor results, thinking that it was due to the prompts and what not, which is partially true. Then I searched for those training on asian faces and it was a pretty consistent story about 50+ training images and 5K-8K steps. Tried that approach and I definitely feel like I'm getting significantly better results in terms of likeness.

I haven't tried yet, but I'm sure this will also be the case for my 2 year old. I was able to cherry pick some good images, but overall most photos didn't look like him. I'm sure if I apply the same number of training images and steps as I did for my wife, the I'll get better results.

In summary, just in my limited experience, few photos and steps hasn't resulted in those mindblowing images you see on Reddit or Youtube, at least in terms of adding customs faces. but I should also mention that I'm just working with txt2img rather than inpainting.
